### PR TITLE
MRG: Tiny improvement to the axis labels in "generalization across time and conditions" decoding example

### PR DESCRIPTION
**Pull request by hoechenberger**
Make it easy to see at a signle glace which condition was used for training and which for testing.

### CODE REVIEW

File: examples/decoding/decoding_time_generalization_conditions.py [hunk 1]
```diff
@@ -75,14 +75,14 @@
...
+fig, ax = plt.subplots()
 im = ax.matshow(scores, vmin=0, vmax=1., cmap='RdBu_r', origin='lower',
                 extent=epochs.times[[0, -1, 0, -1]])
 ax.axhline(0., color='k')
 ax.axvline(0., color='k')
 ax.xaxis.set_ticks_position('bottom')
-ax.set_xlabel('Testing Time (s)')
-ax.set_ylabel('Training Time (s)')
+ax.set_xlabel('"Right" Testing Time (s)')
+ax.set_ylabel('"Left" Training Time (s)')
```
Comment (agramfort) [inline examples/decoding/decoding_time_generalization_conditions.py:85]:
```suggestion
ax.set_xlabel('Condition : "Right"\nTesting Time (s)')
ax.set_ylabel('Condition : "Left"\nTraining Time (s)')
```

**Comment by hoechenberger** [comment]
The x axis label is cropped in the rendered example:
https://output.circle-artifacts.com/output/job/15c958e1-bb3f-44a3-b74d-dcb46b55a6fc/artifacts/0/dev/auto_examples/decoding/decoding_time_generalization_conditions.html




This doesn't happen locally for me. Any idea how to fix that?

**Comment by larsoner** [comment]
We should switch to `constrained_layout=True` figure creation. In 100% of my use cases in the last year or two it has worked without needing any adjustments. We should probably change all our plotting functions to use it at some point...

**Comment by larsoner** [comment]
Looks good to me!

https://output.circle-artifacts.com/output/job/27b04d6a-216f-49df-b3fe-cecb58d044f4/artifacts/0/dev/auto_examples/decoding/decoding_time_generalization_conditions.html#sphx-glr-auto-examples-decoding-decoding-time-generalization-conditions-py

image

I'm assuming the "push right" rather than "fill" is due to 1) the aspect being [forced to be equal in the image by `matshow`](https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.matshow.html) and 2) being associated with the colorbar, which makes sense

**Comment by hoechenberger** [comment]
Great, thanks for your help, @larsoner

**Pull request merged**